src/api/app/assets/javascripts/webui2/package-view_file.js

```diff
@@ -1,31 +1,29 @@
-// FIXME: Refactor this code
 var DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
 var MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
 
+var toTwoChars = function(number, leadingChar) {
+  return (leadingChar + number).slice(-2);
+};
+
 // The jshint ignore:line is needed to ignore the offense: `'addChangesEntryTemplate' is defined but never used.` This is used in the view directly
 function addChangesEntryTemplate() { // jshint ignore:line
   var date = new Date(),
-      day = date.getUTCDate().toString(),
-      hours,
-      minutes,
-      seconds,
-      template;
-
-  if (day.length === 1) { day = " " + day; } // Pad single character day value
-  hours = date.getUTCHours();
-  if (hours < 10) { hours = '0' + hours; }
-  minutes = date.getUTCMinutes();
-  if (minutes < 10) { minutes = '0' + minutes; }
-  seconds = date.getUTCSeconds();
-  if (seconds < 10) { seconds = '0' + seconds; }
+      weekDay = DAYS[date.getUTCDay()],
+      month = MONTHS[date.getUTCMonth()],
+      day = toTwoChars(date.getUTCDate(), ' '),
+      hours = toTwoChars(date.getUTCHours(), '0'),
+      minutes = toTwoChars(date.getUTCMinutes(), '0'),
+      seconds = toTwoChars(date.getUTCSeconds(), '0'),
+      packagerName = $("a.changes-link").data('packagername'),
+      packagerEmail = $("a.changes-link").data('packageremail');
```

Comment:
I would reuse the same result of the finder part, instead of having to traverse the DOM twice.
I mean don't call `$("a.changes-link")` twice, just one time and store it in a variable.